windup/windup-eclipse-plugin
# Fixing WindupServiceTest execution through Tycho command-line

it seems that PDE is now more picky on source that are test or runtime.
So extracted the code that is reused by several test plugin inside a
"runtime" plugin, as opposed to an eclipse-test-plugin

Signed-off-by: Aurélien Pupier <apupier@redhat.com>

TODO:
- add to feature
- test on other machines :-)
- remove no more used (if any) plugin dependencies of core.test plugin
- check other tests that are failing for me later:
```
testStaleIssue(org.jboss.tools.windup.ui.tests.MarkerSyncServiceTest)  Time elapsed: 57.818 s  <<< ERROR!
java.lang.NullPointerException
	at org.jboss.tools.windup.ui.tests.MarkerSyncServiceTest.testStaleIssue(MarkerSyncServiceTest.java:36)

testIssueExplorerPopulated(org.jboss.tools.windup.ui.tests.IssueExplorerTests)  Time elapsed: 5.152 s  <<< FAILURE!
java.lang.AssertionError
	at org.jboss.tools.windup.ui.tests.IssueExplorerTests.testIssueExplorerPopulated(IssueExplorerTests.java:32)

testWindupExecutionIssues(org.jboss.tools.windup.ui.tests.WindupExecutionTest)  Time elapsed: 5.317 s  <<< ERROR!
java.lang.NullPointerException
	at org.jboss.tools.windup.ui.tests.WindupExecutionTest.testWindupExecutionIssues(WindupExecutionTest.java:34)

testWindupExecutionResult(org.jboss.tools.windup.ui.tests.WindupExecutionTest)  Time elapsed: 0.54 s  <<< ERROR!
java.lang.IllegalStateException: Application does not have an active window

testWindupServerStarted(org.jboss.tools.windup.ui.tests.WindupLauncherTest)  Time elapsed: 0.611 s  <<< ERROR!
java.lang.IllegalStateException: Application does not have an active window

testWindupServerStopped(org.jboss.tools.windup.ui.tests.WindupLauncherTest)  Time elapsed: 7.351 s
```
